presto-product-tests/README.md

```diff
@@ -283,9 +283,11 @@ setup outlined below:
 
     ```
     presto-product-tests/conf/docker/singlenode/compose.sh up -d hadoop-master
-    presto-product-tests/conf/docker/singlenode/compose.sh up -d mysql
-    presto-product-tests/conf/docker/singlenode/compose.sh up -d postgres
-    presto-product-tests/conf/docker/singlenode/compose.sh up -d cassandra
+    presto-product-tests/conf/docker/singlenode-mysql/compose.sh up -d mysql
+    presto-product-tests/conf/docker/singlenode-postgresql/compose.sh up -d postgres
+    presto-product-tests/conf/docker/singlenode-cassandra/compose.sh up -d cassandra
+    presto-product-tests/conf/docker/singlenode-sqlserver/compose.sh up -d sqlserver
+    presto-product-tests/conf/docker/singlenode-kafka/compose.sh up -d kafka
```

Comment:
The list of docker testing environments will surely change over time. Maybe instead of listing them all, we could write this in a generic way.

Maybe
```
presto-product-tests/conf/docker/singlenode-*/compose.sh up -d <service>
```
?

Also, I rarely use this this way. I prefer simple:
```
presto-product-tests/conf/docker/singlenode-kafka/compose.sh up -d
```
or even:
```
presto-product-tests/conf/docker/singlenode-kafka/compose.sh up
```